src/elisp/treemacs-branch-creation.el

```diff
@@ -43,13 +43,21 @@ is a marker pointing to POS."
   (copy-marker pos t))
 
 (defsubst treemacs--sort-alphabetic-asc (f1 f2)
-  "Sort F1 and F2 alphabetically asc."
+  "Sort f1 and f2 alphabetically asc."
```

Comment:
Function params must be capitalized or the compiler complains. `make test` can catch these (this stuff will go into a Contributing readme section).